packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { PlusIcon } from '@heroicons/react/24/solid';
```

Comment:
So the heroicons are only used in the stories? Then it should be a dev dependency or optional dependency